erdjs/src/networkStatus.ts

```diff
@@ -0,0 +1,104 @@
+import { IProvider } from "./interface";
+
+/**
+ * An object holding network status configuration parameters.
+ */
+export class NetworkStatus {
+    private static default: NetworkStatus;
+
+    /**
+     * The current round.
+     */
+    public CurrentRound: number;
```

Comment:
Just realised we mix lowerCamelCase with CamelCase notation for public class properties all over.. we should have a refactoring task to have a uniform approach .. these look like "exported" members from golang

Reply:
yep, agree. I'll create a refactoring task